doc/classes/Vector3.xml

```diff
@@ -313,14 +315,15 @@
 			<argument index="0" name="by" type="Vector3">
 			</argument>
 			<description>
-				Returns the vector snapped to a grid with the given size.
+				Returns this vector with each component snapped to the nearest multiple of `step`. This can also be used to round to an arbitrary number of decimals.
```

Comment:
```suggestion
				Returns this vector with each component snapped to the nearest multiple of [code]step[/code]. This can also be used to round to an arbitrary number of decimals.
```